pandas/tests/test_common.py

```diff
@@ -222,3 +224,22 @@ def test_standardize_mapping():
 
     dd = collections.defaultdict(list)
     assert isinstance(com.standardize_mapping(dd), partial)
+
+
+@pytest.mark.parametrize('frame', [
```

Comment:
Since this is either a frame or a series don't use `frame` as the variable name, as that's slightly confusing in case of the former being passed. `obj` should be fine